perf/benchmark/run_linkerd_benchmark_job.sh

```diff
@@ -0,0 +1,96 @@
+#!/bin/bash
+
+# Copyright Istio Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+WD=$(dirname "$0")
+WD=$(cd "$WD"; pwd)
+ROOT=$(dirname "$WD")
+
+# Exit immediately for non zero status
+set -e
+# Check unset variables
+set -u
+# Print commands
+set -x
+
+# This is config for postsubmit cluster.
+export VALUES="${VALUES:-values-istio-postsubmit.yaml}"  # TODO: how to set this for linkerd or we do not need this for linkerd?
```

Comment:
@richardwxn what do you think?

Reply:
We need to make sure that we are providing appropriate CPU / memory.
@richardwxn are these values anything other than cpu/mem limits?